src/global.scss

```diff
@@ -53,7 +53,7 @@ input:-webkit-autofill:active  {
 
 .card-content-md,
 .card-content-ios {
-  color: #333;
+  //color: #333;
```

Comment:
is this supposed to be commented out?

Reply:
yes, I was going to ask, I can't read any text on my local run, with that color